CHANGELOG.md

```diff
@@ -46,7 +50,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `Authenticate()` to handle `POST /auth`
 - Add Personio API v1 client
 
-[Unreleased]: https://github.com/giantswarm/personio-go/compare/v0.4.0...HEAD
+[Unreleased]: https://github.com/giantswarm/personio-go/compare/v0.5.0...HEAD
+[0.5.0]: https://github.com/giantswarm/personio-go/compare/v0.4.0...v0.5.0
```

Comment:
```suggestion
```

Reply:
Maybe revert this, as we need to create the release here, too?

Reply:
I'll revert the changes to CHANGELOG.md